internal/template/funcmap.go

```diff
@@ -28,6 +29,8 @@ var funcMap = map[string]interface{}{
 	"hclField":     hclField,
 	"replace":      replace,
 	"resourceName": resourceName,
+	"now":          time.Now,
```

Comment:
what's the use case for these?

Reply:
It would be useful for generating expiry dates.